apps/skolplattformen-app/ios/Podfile

```diff
@@ -1,7 +1,7 @@
 require_relative '../node_modules/react-native/scripts/react_native_pods'
 require_relative '../node_modules/@react-native-community/cli-platform-ios/native_modules'
 
-platform :ios, '11.0'
+platform :ios, '13.0'
```

Comment:
This might not be needed. I got some input on Discord on how to proceed with iOS.